tests/func/update/network.yml

```diff
@@ -0,0 +1,47 @@
+- name: change item in file for update test
+  lineinfile:
+    dest: "{{ os_migrate_data_dir }}/networks.yml"
+    regexp: "description: ''"
+    line: "    description: 'osm_net_updated'"
+    state: present
+    backup: yes
+    backrefs: yes
+
+- name: scan available networks after before test
+  os_networks_info:
+    filters: "{{ os_migrate_dst_filters }}"
+    auth: "{{ os_migrate_dst_auth }}"
+    auth_type: "{{ os_migrate_dst_auth_type|default(omit) }}"
+    region_name: "{{ os_migrate_dst_region_name|default(omit) }}"
+    validate_certs: "{{ os_migrate_dst_validate_certs|default(omit) }}"
+    ca_cert: "{{ os_migrate_dst_ca_cert|default(omit) }}"
+    client_cert: "{{ os_migrate_dst_client_cert|default(omit) }}"
+    client_key: "{{ os_migrate_dst_client_key|default(omit) }}"
+  register: dst_networks_info_before
+
+- include_role:
+    name: os_migrate.os_migrate.import_networks
+
+- name: scan available networks after update test
+  os_networks_info:
+    filters: "{{ os_migrate_dst_filters }}"
+    auth: "{{ os_migrate_dst_auth }}"
+    auth_type: "{{ os_migrate_dst_auth_type|default(omit) }}"
+    region_name: "{{ os_migrate_dst_region_name|default(omit) }}"
+    validate_certs: "{{ os_migrate_dst_validate_certs|default(omit) }}"
+    ca_cert: "{{ os_migrate_dst_ca_cert|default(omit) }}"
+    client_cert: "{{ os_migrate_dst_client_cert|default(omit) }}"
+    client_key: "{{ os_migrate_dst_client_key|default(omit) }}"
+  register: dst_networks_info_after
+
+- name: ensure description for osm_net changed
+  assert:
+    that:
+      - dst_networks_info_before['openstack_networks'][0].description != None
+      - "dst_networks_info_before['openstack_networks'][0]['description'] \
+         != dst_networks_info_after['openstack_networks'][0]['description']"
```

Comment:
We might even check for the actual value `osm_net_updated` here but not a blocker.